docs/docsite/rst/scenario_guides/guide_meraki.rst

```diff
@@ -1,33 +1,34 @@
 .. _meraki_guide:
 
+******************
 Cisco Meraki Guide
-==================
+******************
 
```

Comment:
If you think the guide will grow, a local TOC might be helpful here. The syntax is:
```
.. contents::
   :local:
```
It's not required, but try it out while you're making the other change and see what you think.